rskj-core/src/main/java/co/rsk/remasc/RemascTransaction.java

```diff
@@ -73,7 +74,7 @@ public long transactionCost(Constants constants, ActivationConfig.ForBlock activ
     }
 
     @Override
-    public RskAddress getSender() {
+    public synchronized RskAddress getSender(ActivationConfig.ForBlock activations, SignatureCache signatureCache) {
```

Comment:
synchronized seems off here

Reply:
It's definitely off for me too. In both remasc and normal transactions